calculator/operations.py

```diff
@@ -1,23 +1,30 @@
+import sys
+from sympy import symbols
+from sympy.plotting import plot as plt
 
+if sys.version_info[0] > 2:
+    from functools import reduce
+    
 def add(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x, y: x+y, args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x, y: x-y, args)
 
 def multiply(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x, y: x*y, args)
 
 def divide(*args):
-    # your implementation here
-    pass
+    return reduce(lambda x, y: float(x)/float(y), args)
 
 def plot(*args):
```

Comment:
@santiagobasulto 
Yes it does
[Plot screenshot](https://github.com/prestonr83/pyp-w1-gw-extensible-calculator/blob/master/plot_sample.png)
[Plot sample output](https://github.com/prestonr83/pyp-w1-gw-extensible-calculator/blob/master/Plot_Output_Sample.txt)